donate.html

```diff
@@ -140,7 +139,7 @@ <h1>Our Story</h1>
         </div>
         <div class="col-md-6">
           <div class="story-image">
-            <img src="img/2019_img.jpg" class="center-block" width="350" height="200">
+            <img src="img/2015_img.jpg" class="center-block">
           </div>
```

Comment:
omg i forgot to tell u!!! jules suggested that the page would look more balanced if the image was the same height as the header and the paragraph. so it would be like a row with two col-6's in it. the first col-6 would be the header/paragraph and the second would just be the picture. and the same would go for the donor-promise section